jwst/flatfield/flat_field_step.py

```diff
@@ -46,6 +46,8 @@ def process(self, input):
             self.log.debug('Input is a CubeModel')
         elif isinstance(input_model, datamodels.ImageModel):
             self.log.debug('Input is an ImageModel')
+        elif isinstance(input_model, datamodels.IFUImageModel):
```

Comment:
There must be a simpler, more elegant way to do this whole if-else block. Rather than testing for individual instances of a particular datamodel type, there must be a way to just print what the data model type is from some attribute value somewhere within the datamodel.